code/modules/projectiles/guns/ballistic/launchers.dm

```diff
@@ -3,7 +3,7 @@
 
 /obj/item/gun/ballistic/revolver/grenadelauncher
 	desc = "A break-operated grenade rifle. Projectiles travel slowly."
-	name = "grenade rifle"
+	name = "M79 grenade launcher"
```

Comment:
Thought we were doing away with the realistic gun name convention.

Reply:
:(